website/docs/reference/commands/test.md

```diff
@@ -28,4 +28,4 @@ dbt test --select "one_specific_model,test_type:singular"
 dbt test --select "one_specific_model,test_type:generic"
 ```
 
-For more information on writing tests, see the [Testing Documentation](/docs/build/tests).
+For more information on writing tests, see the [Testing Documentation](/docs/build/data-tests).
```

Comment:
Will want to come back to this page when we add all of the content for unit testing!